logic.js

```diff
@@ -0,0 +1,76 @@
+let Twitter = function() {
+
+    const posts = [{
+            text: "First post!",
+            id: "p1",
+            comments: [
+                { id: "c1", text: "First comment on first post!" },
+                { id: "c2", text: "Second comment on first post!!" },
+                { id: "c3", text: "Third comment on first post!!!" }
+            ]
+        },
+        {
+            text: "Aw man, I wanted to be first",
+            id: "p2",
+            comments: [
+                { id: "c4", text: "Don't wory second poster, you'll be first one day." },
+                { id: "c5", text: "Yeah, believe in yourself!" },
+                { id: "c6", text: "Haha second place what a joke." }
+            ]
+        }
+    ]
+
+    let postIdCounter = 2;
+    let commentIdCounter = 7;
+
+    const getPosts = function() {
+        return posts
+    }
+
+    const addPost = function(text) {
+        posts.push({
+            text: text,
+            id: 'p' + ++postIdCounter,
+            comments: []
+        })
+    }
+
+    const removePost = function(postId) {
+        for (let i = 0; i < posts.length; i++) {
+            if (posts[i].id === postId) {
+                posts.splice(i, 1);
+                console.log(posts)
```

Comment:
after finishing your code, please get rid of all the unnecessary console.logs, comments and empty lines